pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -314,6 +314,12 @@ sudoPrompt:
   explanation: 'This will modify the following paths:'
   buttonText: OK
 
+unmetPrerequisites:
+  title: Rancher Desktop is unable to start
+  message: 'Rancher Desktop cannot start because requirements are missing or not configured:'
+  action: 'Please ensure all requirements are met and try again. Rancher Desktop will now close'
```

Comment:
This is a bit of a nitpick, but I think it would be good to have a period on the end of "Rancher Desktop will now close".

Reply:
period added.  squash will happen during the merge.